sonarqube/content.md

```diff
@@ -0,0 +1,43 @@
+# What is SonarQube?
+
+SonarQube is an open source platform for continuous inspection of code quality.
+
+> [wikipedia.org/wiki/SonarQube](http://en.wikipedia.org/wiki/SonarQube)
+
+%%LOGO%%
+
+# How to use this image
+
+## Run SonarQube
+
+The server is started this way:
+
+    docker run -d --name sonarqube -p 9000:9000 -p 9092:9092 sonarqube:5.1
+
+To analyse a project:
+
+    $ On Linux:
+    mvn sonar:sonar
+
+    $ With boot2docker:
+    mvn sonar:sonar -Dsonar.host.url=http://$(boot2docker ip):9000 -Dsonar.jdbc.url="jdbc:h2:tcp://$(boot2docker ip)/sonar"
```

Comment:
On this and the other code block sections can we swap the indent to a tab rather than spaces.  Also, make line 21 be indented with a tab as well.  TravisCI should have caught these, but for some reason is not running on this PR.  After that, LGTM.

(a `git add [files]`, `git commit --amend`, and `git push -f` will be the easiest, and feel free to ping us once you have it updated)